projects/app/src/data/questions/util.ts

```
@@ -0,0 +1,61 @@
+import { invariant } from "@haohaohow/lib/invariant";
+import type { DeepReadonly } from "ts-essentials";
+
+export function hanziOrPinyinWordCount(
+  choice: OneCorrectPairQuestionChoice,
+): number {
+  switch (choice.kind) {
+    case `hanzi`: {
+      return characterCount(choice.value);
+    }
+    case `pinyin`: {
+      return typeof choice.value === `string` ? 1 : choice.value.length;
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)